.npmignore

```diff
@@ -3,9 +3,8 @@ node_modules
 config
```

Comment:
It would be better specify directory adding slash at the end.
Also node_modules is ignored by default.
https://docs.npmjs.com/misc/developers#keeping-files-out-of-your-package